### PR TITLE
Set up a "Hello Microverse" project

### PR DESCRIPTION
**Pull request by VivekRaaZ**
In this project milestone I:

1. Created an index.html file
2. Created a style.css file
3. Created a Readme file
4. Added Linters to the project

### CODE REVIEW

**Comment by VivekRaaZ** [comment]
This is a simple html and css files of "Hello Microverse" project.

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @VivekRaaZ , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
close

 ## Highlights

- [ ] Linters set-up ✔️
- [ ] Commit messages descrptive ✔️
- [ ] Professional Github Flow used ✔️



## Required Changes ♻️
_Check the comments under the review._

- [ ] Good job so far, 👏  However, it appears that you have not used the microverse Readme file and as part of the requirement, you are supposed to use this microverse [template](https://github.com/microverseinc/readme-template) and update the contents according to your project. Please know that the only section that is allowed to be removed is the FAQ section because that is `OPTIONAL` 🙂 
- [ ] Please ensure that you used a descriptive Pull Request summary as it shows on yours as `No Description` :slightly_smiling_face: 

Below is an example of a good PR summary :arrow_down: 

PR Summary


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by VivekRaaZ** [comment]
I have updated all. would you like to recheck please.

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @VivekRaaZ, 



Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Highlights
✔️ Good Effort
✔️ Your design is good.
✔️ All linters passed

 ## Required Changes ♻️
- [ ] As mentioned by the previous reviewer :point_up_2:  however, it appears that you have not used the microverse Readme file and as part of the requirement, you are supposed to use this microverse [template](https://github.com/microverseinc/readme-template) and update the contents according to your project. Please know that the only section that is allowed to be removed is the FAQ section because that is `OPTIONAL` but still you didn't push your README file so kindly add microverse provided README file :slightly_smiling_face:

**_Check below :point_down:  your README file which is not a microverse-provided README file_**

Screenshot from 2023-07-06 20-33-40


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by VivekRaaZ** [comment]
sir my README.md file in main branch. please check it.

**Comment by aamirkhan2478** [comment]
> sir my README.md file in main branch. please check it.

Please push your README file in that branch and then submit it again

**Comment by VivekRaaZ** [comment]
i have update README.md file. does its fine or i have to make some changes more ?

please give me feedback, i have only remain one review.

File: README.md [hunk 1]
```diff
@@ -1 +1,161 @@
...
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> I would like to thanks microverse team, github team and my coding partners, who contributed towards the completion of this task.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (shahadat3669) [inline README.md]: Well done on your progress so far! However, there are a few adjustments that still need to be made:

1. The License link in your README file is currently broken. It's important to provide a working link that leads visitors to the customized License file. Please update your README file to include the correct link.

[VivekRaaZ-R-Linters-at-html-css-branch.webm](https://github.com/VivekRaaZ/R-Linters/assets/55840999/4c809981-6dbb-4694-a69e-b6f94bc8b380)

2. I noticed that you do not have a customized License file in your project. It is essential to have a proper License file that reflects the specific licensing terms of your project. You can use the [Readme Template](https://github.com/microverseinc/readme-template/tree/master) as a reference and replace all the placeholders with meaningful details to create your customized License file based on the MIT License.

[microverseinc-readme-template-A-README-template-that-can-be-used-to-give-any-project-clear-documentation-.webm](https://github.com/VivekRaaZ/R-Linters/assets/55840999/74cf6f83-dcac-44cc-9f39-ae6bd7bf34d2)

Please make these adjustments to ensure that your project adheres to the necessary licensing requirements and provides clear information to users. Great job so far, and keep up the good work!

File: README.md [hunk 2]
```diff
@@ -1 +1,161 @@
...
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
```
Comment (shahadat3669) [inline README.md]: Great job on the Readme file! There is one improvement that can be made to align with the content:

- [ ] Please remove "FAQ (OPTIONAL)" from the "Table of Contents" section, as there is no section existing in your Readme file with this name. Keeping the table of contents accurate will ensure readers can navigate the document effectively. 🙏 

By making this adjustment, the table of contents will reflect the sections in the Readme file. Well done on your progress! 👍

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi @VivekRaaZ , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
✅ Good PR Title
✅ Good PR Description
✅ Good use of GitHub Flows


 ## Required Changes ♻️

- [ ] 🔍 The last commit you made did not check and successfully pass the linter errors. It's crucial to ensure that your code adheres to the linting rules and conventions. Please update your commit and the previous commit to check and pass the linters. This will help maintain clean and consistent code throughout your project.

image

Take the time to review your code and address any linting errors or warnings. This will improve the overall quality and readability of your codebase. Keep up the good work! 👍

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by VivekRaaZ** [comment]
I have just done all changes according to reviewer, please check it. @shahadat3669

**Review by skyv26 — APPROVED** [review]
## Hi @VivekRaaZ, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Professional `README` template ✅ 
- Linters are green ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**